truss/contexts/local_loader/docker_build_emulator.py

```diff
@@ -41,18 +43,20 @@ def _resolve_values(keys: List[str]) -> List[str]:
 
         result = DockerBuildEmulatorResult()
         for cmd in self._commands:
-            if cmd.cmd not in ["ENV", "ENTRYPOINT", "COPY", "WORKDIR"]:
+            if not cmd.is_supported:
                 continue
             values = _resolve_values(cmd.value)
-            if cmd.cmd == "ENV":
+            if cmd.instruction == DockerInstruction.ENV:
+                if "=" in values[0]:
+                    values = values[0].split("=", 1)
                 result.env[values[0]] = values[1]
-            if cmd.cmd == "ENTRYPOINT":
+            if cmd.instruction == DockerInstruction.ENTRYPOINT:
                 result.entrypoint = list(values)
-            if cmd.cmd == "COPY":
+            if cmd.instruction == DockerInstruction.COPY:
```

Comment:
Mentioned in the PR description, but from my [reading](https://github.com/basetenlabs/truss/blob/9730a0914f524d07e2574061d54b21c217709dc0/truss/tests/conftest.py?plain=1#L700) it seems like the COPY command is the only one that can actually have a side effect that impacts tests?